### PR TITLE
Update Logo

### PR DESCRIPTION
**Pull request by mohamednajiub**
- Update Favicon
- Update Header Logo
- Update Footer Logo

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ const siteConfig = {
   users,
 
   /* path to images for header/footer */
-  headerIcon: 'img/favicon.ico',
-  footerIcon: 'img/favicon.ico',
+  headerIcon: 'img/logo_icon.svg',
+  footerIcon: 'img/logo_icon.svg',
   favicon: 'img/favicon.ico',
```
Comment (elharony) [inline website/siteConfig.js:55]: Thanks for adding it, @mohamednajiub !
Can you add the `Favicon` here as well?

Comment (mohamednajiub) [reply website/siteConfig.js:55]: I replaced the old favicon with the new favicon and kept for its name in the file so if you opened the favicon file you can see the new favicon.
You can see that there is changes in favicon.ico file.

Comment (islamCodehood) [reply website/siteConfig.js:55]: I reviewed the file favicon.ico and it is the new favicon.

**Review by islamCodehood — APPROVED** [review]
Approved. 
Thank you so much for the sincere efforts!

**Pull request merged**